index.html

```diff
@@ -11,6 +11,1053 @@
+                  <a href="#">Opinion</a>
+                </h3>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+              <div class="mid-card">
+                <div class="card" style="width: 18rem;">
```

Comment:
- [ ] Remove this inline styling, this is one of the things affecting the horizontal scroll on your page.